flink-connectors/flink-sql-connector-elasticsearch7/pom.xml

```diff
@@ -63,14 +63,10 @@ under the License.
 								</includes>
 								<excludes>
 									<!-- These dependencies are not required. -->
-									<exclude>com.carrotsearch:hppc</exclude>
```

Comment:
Do we need to include `carrotsearch`? I saw elasticsearch connector 6 exclude this.

Reply:
yes，elasticsearch connector 7 need this.